src/hints/testLessThan.test.ts

```diff
@@ -0,0 +1,66 @@
+import { describe, expect, test } from 'bun:test';
+
+import { Felt } from 'primitives/felt';
+import { Register } from 'vm/instruction';
+import { VirtualMachine } from 'vm/virtualMachine';
+import { OpType } from 'hints/hintParamsSchema';
+import { HintName } from 'hints/hintName';
+import { testLessThanParser } from './testLessThan';
+
+const TEST_LESS_THAN = {
+  TestLessThan: {
+    lhs: {
+      Deref: {
+        register: 'AP',
+        offset: 0,
+      },
+    },
+    rhs: {
+      Immediate: '0x100000000',
+    },
+    dst: {
+      register: 'AP',
+      offset: -1,
+    },
+  },
+};
+
+describe('TestLessThan', () => {
+  test('should properly parse TestLessThan hint', () => {
+    const hint = testLessThanParser.parse(TEST_LESS_THAN);
+
+    expect(hint).toEqual({
+      type: HintName.TestLessThan,
+      lhs: {
+        type: OpType.Deref,
+        cell: {
+          register: Register.Ap,
+          offset: 0,
+        },
+      },
+      rhs: {
+        type: OpType.Immediate,
+        value: new Felt(BigInt('0x100000000')),
+      },
+      dst: {
+        register: Register.Ap,
+        offset: -1,
+      },
+    });
+  });
+
+  test.each([
+    [new Felt(-2n), new Felt(0n)],
+    [new Felt(2n), new Felt(1n)],
+  ])('should properly execute TestLessThan hint', (lhs, expected) => {
```

Comment:
```suggestion
  test.each([
    [new Felt(BigInt('0x100000000')), new Felt(1n)],
    [new Felt(BigInt('0x100000001')), new Felt(0n)],
    [new Felt(-2n), new Felt(0n)],
    [new Felt(2n), new Felt(1n)],
  ])('should properly execute TestLessThan hint', (lhs, expected) => {
```

Reply:
It is a strict inequality so `'0x100000000'` should return False (`0n`), but having cases around the bound should definitely be added. I'll add 0x100000000 and 0x0ffffffff